crates/katana/primitives/src/genesis/json.rs

```diff
@@ -216,6 +216,7 @@ pub struct GenesisJson {
     pub accounts: HashMap<ContractAddress, GenesisAccountJson>,
     #[serde(default)]
     pub contracts: HashMap<ContractAddress, GenesisContractJson>,
+    pub gather_from_block: BlockNumber,
```

Comment:
i think the genesis is determined by the smart contract in the underlying settlement layer, it also indicates the minimum starting point of when messages should be collected from. so it's more appropriate to rename this field into something like `settlement_block_number` which implicitly tells the messaging task to start fetching the messages starting from this block number of the L1.

Reply:
sure